src/app/modules/analytics-config/services/analytics-import/analytics-import.service.ts

```diff
@@ -533,6 +538,86 @@ export class AnalyticsImportService {
     return widgetData;
   }
 
+  private getMetricsTableWidgetData(component: AnalyticComponentConfig, contributor: ContributorConfig) {
+    console.log('getMetricsTableWidgetData');
+    const widgetData = this.metricsTableFormBuilder.build(contributor.collection);
+    const dataStep = widgetData.customControls.dataStep;
+    const renderStep = widgetData.customControls.renderStep;
+    const uuid = widgetData.customControls.uuid;
+    const usage = widgetData.customControls.usage;
+    const title = widgetData.customControls.title;
+    contributor.configuration.forEach(config => {
+      const subtable = this.metricsTableFormBuilder.buildSubTable(config.collection);
+      subtable.customControls.aggregationField.setValue(config.termfield);
+      config.metrics.forEach(m => {
+        const column = this.metricsTableFormBuilder.buildSubTableColumn(config.collection);
+        column.customControls.metricCollectFunction.setValue(m.metric);
+        column.customControls.metricCollectField.setValue(m.field);
+        column.customControls.sort.setValue('');
+        if (!!contributor.sort) {
```

Comment:
This is to check if the sort of the conributor matches the one of the component ?